netlify.toml

```
@@ -0,0 +1,4 @@
+[build]
+  command = "jlpm install && jlpm build"
+  publish = "dist"
+  environment = { PYTHON_VERSION = "3.13" }
```

(nit: no newline at end of file)

```suggestion
  environment = { PYTHON_VERSION = "3.13" }

```